contracts/ERC20.sol

```diff
@@ -101,12 +101,12 @@ contract ERC20 is IERC20 {
         _approve(owner_, spender_, amount_);
     }
 
-    function transfer(address recipient_, uint256 amount_) external override returns (bool success_) {
+    function transfer(address recipient_, uint256 amount_) public virtual override returns (bool success_) {
```

Comment:
Hmm, should these all be public?

Reply:
`external` would require us to overwrite when we override and we could not call `super`